docs/csharp/language-reference/operators/lambda-expressions.md

```diff
@@ -204,12 +204,78 @@ customers.Where(c => c.City == "London");
 The general rules for type inference for lambdas are as follows:
 
 - The lambda must contain the same number of parameters as the delegate type.
-
 - Each input parameter in the lambda must be implicitly convertible to its corresponding delegate parameter.
-
 - The return value of the lambda (if any) must be implicitly convertible to the delegate's return type.
   
-Note that lambda expressions in themselves don't have a type because the common type system has no intrinsic concept of "lambda expression." However, it's sometimes convenient to speak informally of the "type" of a lambda expression. In these cases the type refers to the delegate type or <xref:System.Linq.Expressions.Expression> type to which the lambda expression is converted.
+## Natural type for lambda expressions
+
+Lambda expressions in themselves don't have a type because the common type system has no intrinsic concept of "lambda expression." However, it's sometimes convenient to speak informally of the "type" of a lambda expression.That informal "type" refers to the delegate type or <xref:System.Linq.Expressions.Expression> type to which the lambda expression is converted.
```

Comment:
```suggestion
Lambda expressions in themselves don't have a type because the common type system has no intrinsic concept of "lambda expression." However, it's sometimes convenient to speak informally of the "type" of a lambda expression. That informal "type" refers to the delegate type or <xref:System.Linq.Expressions.Expression> type to which the lambda expression is converted.
```